pkg/broker/types.go

```diff
@@ -10,6 +10,11 @@ type CatalogResponse struct {
 // ProvisionResponse is sent as the response to a provision call.
 type ProvisionResponse struct {
 	osb.ProvisionResponse
+
+	// AlreadyCreated - is used to if the request was already completed
+	// and the requested parameters are identical to the existing
+	// Service Instance.
+	AlreadyCreated bool `json:"-"`
```

Comment:
I would suggest just `Created` or `Exists` instead of `AlreadyCreated`. WDYT?

Reply:
I am fine with `Exists`